spec/controllers/guides_controller_spec.rb

```diff
@@ -67,4 +67,12 @@
       end
     end
   end
+
+  describe "a malicious user trying to initialise an unwanted Guide STI constant" do
```

Comment:
I added this test to drive adding `params[:type].presence_in(%w{ GuideCommunity Point })` which is defending against someone trying to slip something into `Guide.new(type: type)` which, deep deep down, is essentially performing an eval on params 😱 

It's probably too paranoid - I kind of the like the paranoia here though. Evalling params scares me!